python/llm/src/bigdl/llm/langchain/embeddings/transformersembeddings.py

```diff
@@ -83,7 +85,18 @@ def from_model_id(
         model_kwargs: Optional[dict] = None,
         **kwargs: Any,
     ):
-        """Construct object from model_id"""
+        """
+        Construct object from model_id.
+
+        Args:
+
+            model_id: Path for the huggingface repo id to be downloaded or the huggingface
+                      checkpoint folder.
+            model_kwargs: Keyword arguments that will be passed to the model and tokenizer.
+            kwargs: Extra arguments that will be passed to the model and tokenizer.
+        
+        Returns: Instances of the embedding model and tokenizer.
```

Comment:
will this return two objects?

Reply:
I thinks so, since it will return hf model and hf tokenizer objects.

Reply:
Really? In line 124, it will return cls(...), which is a TransformersEmbeddings object, no?